client-lib/src/main/java/com/ibm/ws/repository/exceptions/RepositoryResourceNoConnectionException.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+* Copyright (c) 2017 IBM Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+package com.ibm.ws.repository.exceptions;
+
+/**
+ * This is thrown when an operation is attempted on a resource that requires a connection and the resource does not have a connection.
+ */
+public class RepositoryResourceNoConnectionException extends RepositoryResourceException {
+
+    private static final long serialVersionUID = 8976559783458635686L;
```

Comment:
Do we need a real version id? I kind of prefer to have the default one here, as it indicates that no one has actually thought about whether this is serializable or not.